src/CmsFileManagerBundle/objects/TPkgCmsFileManager_FileSystem.class.php

```diff
@@ -112,7 +112,11 @@ public function mkdir($path, $mode = 0777, $recursive = false)
             trigger_error('File permissions are handled by the administrator. Do not try to set the mode by hand.', E_USER_DEPRECATED);
         }
 
-        return mkdir($path, 0777, $recursive);
+        try {
+            return mkdir($path, 0777, $recursive);
+        } catch (Exception $e) {
+            throw new Exception(sprintf("Cannot create folder %s. Error: %s", $path, $e->getMessage()), $e->getCode(), $e);
```

Comment:
The part "Error %s" is not needed here.

If interesting/interested this is part of the parent exception.

Reply:
okay, fixed.